### ISSUE TITLE
Cannot customize IntermediateOutputPath and BaseIntermediateOutputPath for C# UAP test project

### ISSUE THREAD

**Issue by shawnhar** [Microsoft/vstest#1535]
My original repro:
- Clone https://mscodehub.visualstudio.com/_git/Win2D
- Edit winrt\test.managed\winrt.test.managed.uap.csproj
- From line 27, delete:
```
    <IntermediateOutputPath>obj</IntermediateOutputPath>
    <BaseIntermediateOutputPath>obj</BaseIntermediateOutputPath>
```
- Using Visual Studio 15.6.4
- Load win2d.uap.sln
- Attempt to run test.managed unit tests
- Tests cannot launch because MSTest.TestAdapter was not copied to the build output folder

Simplified repro:
- File / New / Project / Visual C# / Windows Universal / Unit Test App (Universal Windows)
- Test Explorer / Run All
  - This works
- Close the solution
- Delete the bin and obj folders
- Edit UnitTestProject1.csproj (eg. in notepad)
- At line 21 (inside the first <PropertyGroup>) add:
```
     <IntermediateOutputPath>foo</IntermediateOutputPath>
     <BaseIntermediateOutputPath>foo</BaseIntermediateOutputPath>
```
- Reopen the solution in Visual Studio
- Test Explorer / Run All
```
  [4/6/2018 1:37:02 PM Informational] ------ Run test started ------
  [4/6/2018 1:37:03 PM Warning] No test is available in entrypoint\UnitTestProject1.exe. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
  [4/6/2018 1:37:03 PM Informational] ========== Run test finished: 0 run (0:00:01.3680073) ==========
```

**Comment by mayankbansal018** [comment]
@shawnhar , the issue was on nuget side, which has been fixed in update 15.7. It will be released around May 7th.

**Comment by mayankbansal018** [comment]
@shawnhar , I'm closing this issue, as it is fixed in latest 15.7 preview, & will be available in official 15.7 RTM
